queries/markdown/highlights.scm

```diff
@@ -34,6 +34,29 @@
   (info_string (language) @conceal
   (#set! conceal "")))
 
+;; Conceal blockquote markers
+((block_quote_marker) @punctuation.special
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  (#eq? @punctuation.special ">")
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  (#eq? @punctuation.special "> ")
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  ; for indented code blocks
+  (#eq? @punctuation.special ">     ")
```

Comment:
Why 4, what if it's 2, 3, ...

Reply:
Due to the way the parser is implemented, there can never be such a case. The only `block_continuation` marker contents are `>`, `> `, and `>     ` (if there is an indented code block). Any variable amount of spaces other than these are not captured in the `block_continuation` group, and thus they work nicely here. For example, in:
```md
>   block continuation with three spaces
```
The first `> ` (caret with one space) will be captured under the block continuation node, and the rest of the spaces are captured under the next `paragraph` node.